src/pages/App.tsx

```diff
@@ -58,10 +80,31 @@ const Footer = styled.div`
   text-align: center;
   width: 420px;
   margin: auto;
+  padding-bottom: 16px
   color: #fffa;
 `
 
 export default function App() {
+  const { library } = useActiveWeb3React()
+  const [currentMarquee, setCurrentMarquee] = useState('')
+  const handleMarqueeUpdate = (newMarquee: string) => {
+    setCurrentMarquee(newMarquee)
+  }
+
+  useEffect(() => {
+    if (library) {
+      const contract = new ethers.Contract(MARQUEE_CONTRACT_ADDRESS, marqueeAbi, library)
+      contract
+        .marquee()
+        .then((marquee: string) => {
+          setCurrentMarquee(marquee)
+        })
+        .catch((error: Error) => {
+          console.error('Error retrieving marquee:', error)
+        })
+    }
+  }, [library])
+
```

Comment:
Maybe we dont need here and can go to insde the Marquee Component as well.

Reply:
I had trouble figuring out how to get the sign to update from the Marquee component. If we can have it update there that would be awesome.